pxr/imaging/plugin/hdRpr/houdini/CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
+if(RPR_ENABLE_OPENVDB_SUPPORT)
```

Comment:
IMO RPR_ENABLE_OPENVDB_SUPPORT should always be true if Houdini build no? 

Only usdview should have optional openvdb

Reply:
I have enabled OpenVDB by default when building Houdini plugin in abed8b0
But I left possibility for the user to disable it

Reply:
Ok, not sure why they'd want to but that makes sense.